symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/ViewOrganizationGeneralInformation.vue

```diff
@@ -44,11 +44,14 @@
               />
             </oxd-grid-item>
             <oxd-grid-item>
-              <oxd-input-field
-                label="Number of Employees"
-                v-model="organization.noOfEmployees"
-                :disabled="true"
-              />
+              <div>
+                <oxd-label class="no-of-employees-label">
```

Comment:
@KaveeshaSilva1996 No need to use custom elements. we can utilise OXD components.

```
<oxd-grid-item>
  <oxd-input-group label="Number of Employees">
    <oxd-text tag="h5">
      {{ organization.noOfEmployees }}
    </oxd-text>
  </oxd-input-group>
</oxd-grid-item>
```